source/agora/consensus/EnrollmentManager.d

```diff
@@ -823,6 +823,33 @@ public class EnrollmentManager
         return PublicKey(this.key_pair.V[]);
     }
 
+    /***************************************************************************
+
+        Get the index of this node's enrollment in the sorted enrolled UTXOs
+
+        Returns:
+            the index of the enrollment, or ulong.max if this node is not
+            enrolled as a validator.
+
+    ***************************************************************************/
+
+    public ulong getIndexOfEnrollment () @safe nothrow
+    {
+        if (this.enroll_key == Hash.init)
+            return ulong.max;
+
+        Hash[] utxo_keys;
+        assert(this.validator_set.getEnrolledUTXOs(utxo_keys));
```

Comment:
```
if (!this.validator_set.getEnrolledUTXOs(utxo_keys));
    assert(0);
```

Because currently the assert you wrote here will disappear in `-release` mode, so the function call will not exist.

Reply:
assert(0) is a special-case: It's always there, even in `-release` mode.

Reply:
I got it. Will update.
Thanks.